lib/pbench/test/unit/server/test_requests.py

```diff
@@ -90,7 +90,7 @@ class TestUpload:
     filetree_create_path = None
     tarball_deleted = None
 
-    @pytest.fixture
```

Comment:
Can we move this kind of change to a separate PR where we apply it everywhere else it is needed?